dev/devicelab/manifest.yaml

```diff
@@ -549,11 +549,11 @@ tasks:
     stage: devicelab_ios
     required_agent_capabilities: ["mac/ios"]
 
-  # post_backdrop_filter_perf_ios__timeline_summary:
-  #   description: >
-  #     Measures the runtime performance of animations after a backdrop filter is removed on iOS.
-  #   stage: devicelab_ios
-  #   required_agent_capabilities: ["mac/iphonexs"]
+   post_backdrop_filter_perf_ios__timeline_summary:
+     description: >
+       Measures the runtime performance of animations after a backdrop filter is removed on iOS.
+     stage: devicelab_ios
+     required_agent_capabilities: ["mac/iphonexs"]
```

Comment:
I think we no longer have `iphonexs` in the devicelab. We probably should just use `mac/ios`.

Reply:
Let's also add a `flaky: true` here just to be cautious.

Reply:
Done